api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,126 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user) who share access to resources in Microsoft services or in your app. Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs. All operations in Microsoft Graph on groups require administrator consent.

```

Comment:
Maybe just add "and sometimes other principals" to your first sentence? i.e. "Groups are collections of [users](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user), and, sometimes, other principals, that share access to resources in Microsoft services or in your app. " --- although the other suggestions work too. Only thing about saying "other resources" is that it is a bit open ended -- and we use resources later in the sentence. 

The last sentence about admin consent should be moved to an Authorization section, similar to what we do for AAD overview: https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/azure_ad_overview#authorization . For most things you need Group permissions, for some APIs you may also need Directory or User permissions as well (I don't think you need to go into detail.) Group and Directory permissions always require Administrator consent. With delegated permissions, for some operations on groups the signed-in user may need to be a group owner and for others they may need to be a member of an appropriate administrator role. 

Just FYI, I don't think you need to go into  detail in the first sentence, but I think the group breakdown is something like this:
O365 Groups: users (not sure about contacts)
Security groups: users, service principals, other security groups.
Distribution groups: users, contacts, mail-enabled security groups, other distribution groups

The AAD Graph API reference docs are pretty specific about what resources can be members of security and distribution groups.